pkg/operator/controller/ingress/deployment_test.go

```diff
@@ -2433,7 +2548,7 @@ func TestDesiredRouterDeploymentRouterExternalCertificate(t *testing.T) {
 		t.Error(err)
 	}
 
-	deployment, err = desiredRouterDeployment(ic, ingressControllerImage, ingressConfig, infraConfig, apiConfig, networkConfig, false, false, nil, clusterProxyConfig, true)
+	deployment, err = desiredRouterDeployment(ic, ingressControllerImage, ingressConfig, infraConfig, apiConfig, networkConfig, false, false, nil, clusterProxyConfig, true, false)
```

Comment:
There are no tests where the feature is `true`.  I think we need one.  Also, I'm curious what happens when `unsupportedConfigOverrides.DynamicConfigManager` is an empty string.